corehq/apps/app_manager/models.py

```diff
@@ -1135,7 +1135,7 @@ def validate_form(self):
             form.strip_vellum_ns_attributes()
             try:
                 if form.xml is not None:
-                    validate_xform(self.get_app().domain, etree.tostring(form.xml))
+                    validate_xform(self.get_app().domain, etree.tostring(form.xml, encoding="unicode"))
```

Comment:
Not your problem but I have to say... `encoding="unicode"` is a bit jarring because _Unicode is NOT an encoding_. Anyway, this is the right way to do it.